setup.py

```diff
@@ -21,6 +21,8 @@
         "tox==3.21.1",
         "pre-commit==2.9.3",
         "apache-airflow==2.0.1",
+        "apache-airflow-providers-cncf-kubernetes==1.1.0",
+        "kubernetes==12.0.1",
```

Comment:
Seems is like a `kubernetes` is a depenedency of `..cncf-kubernetes` - I think we may drop this one.

Reply:
Well noticed, dropping kubernetes dependency then.